### DISCUSSION TITLE
Normalize beatmap stats

### DISCUSSION THREAD

**Discussion by MitsukaiDokuro** [ppy/osu#13132] [Ideas]
**Describe the new feature:** Switch beatmap stats to a natural logarithmic scale. It makes a lot more sense and just feels intuitive.

**Proposal designs of the feature:** 
Example for AR (Approach Rate) with `minValue = 1800`, `midValue = 1200`, `maxValue = 450`
- old formula: `approach time [ms] = AR > 5 ? midValue + (maxValue - midValue) * (AR - 5) / 5 : midValue - (midValue - minValue) * (5 - AR) / 5`
- new formula: `approach time [ms] = minValue* exp(AR * (1 / 10) * ln(maxValue / minValue))` (reverse would be `AR = 10 * ln(approach time [ms] / minValue) / ln(maxValue / minValue)`)

This should work for approach time, spinner difficulty, accuracy thresholds, hp drain rate. Circle size doesn't need a change IMO.

**Comment by peppy** [comment]
While I can understand why this would be better, the question of historical context comes into play.

Every existing post, score, and user's understanding of numbers will no longer make sense. We would have to start referring to new-AR and old-AR or similar.

I'm not sure this is the kind of change we want to make at *any* point, but now is definitely not the time.

**Comment by MitsukaiDokuro** [comment]
I totally understand the confusion caused by this change and that getting the new client done is top priority. Maybe at a later point of time there will be an opportunity to offer new-Stats as an alternative display form and see how the community reacts. Thanks for your time.
